src/product-resource.js

```diff
@@ -118,6 +119,25 @@ class ProductResource extends Resource {
       .then(defaultResolver('products'))
       .then(paginateProductConnectionsAndResolve(this.graphQLClient));
   }
+
+  /**
+   * Find recommended products related to a given product_id. To learn more about how recommendations are generated, see Showing product recommendations on product pages.
```

Comment:
`productId` instead of `product_id`?

Maybe we should include a link to https://shopify.dev/themes/product-merchandising/recommendations in here as well? Or instead of "see Showing product..."

Reply:
yeah, make sense, took this comment from docs. Will add some docs link instead as you suggested